EvidenceApi.Tests/V1/E2ETests/BaseControllerTests.cs

```diff
@@ -0,0 +1,105 @@
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NUnit.Framework;
+
+namespace EvidenceApi.Tests.V1.E2ETests
+{
+    public class BaseControllerTests : IntegrationTests<Startup>
+    {
+        [Test]
+        public async Task CanCreateAuditEventForGetRequestWithoutQueryParameters()
+        {
+            var documentSubmission = TestDataHelper.DocumentSubmission(true);
+            DatabaseContext.DocumentSubmissions.Add(documentSubmission);
+            DatabaseContext.SaveChanges();
+            var uri = new Uri($"/api/v1/document_submissions/{documentSubmission.Id}", UriKind.Relative);
+            Client.DefaultRequestHeaders.Remove("UserEmail");
```

Comment:
You could extract `Client.DefaultRequestHeaders.Remove("UserEmail")` into a `[SetUp]` method because it's required for every test

Reply:
👍